### DISCUSSION TITLE
How to run in Linux?

### DISCUSSION THREAD

**Discussion by Cazra** [icsharpcode/ILSpy#3100] [Help]
I'm sorry if this is a stupid question, but how do I run ILSpy to disassemble .NET executables in a Linux environment via a terminal? I followed the build instructions for Unix/Mac in the README to build the project... Now what?

I'm not at all familiar with the use of the .Net SDK or PowerShell.

**Comment by niravx23** [comment]
Maybe , Have a look in [ILSPYCmd](https://github.com/icsharpcode/ILSpy/blob/master/ICSharpCode.ILSpyCmd/README.md).
